source/users/custom-metadata.txt

```diff
@@ -713,6 +733,18 @@ Configure Authentication Provider Metadata
          fields, App Services returns a ``204`` response.
 
 
+
+.. note:: Tips for Users with Multiple Linked Authentication Providers  
+
+  - To ensure the most up-to-date metadata, users should re-authenticate after 
+    switching to a different authentication provider. Failure to do so may result 
+    in outdated metadata reflected in the :guilabel:`Users` table in 
+    the UI's :guilabel:`App Users` page and for requests using authentication providers.
+  - If a user switches between authentication providers, there may be a 30 minute delay 
+    in propagating metadata to the :guilabel:`Users` table and for requests using 
```

Comment:
Minor suggestion for concision and avoid internal implementation detail that's not relevant to users.
```suggestion
  - If a user switches between authentication providers, it may take 30 minutes to
     to propagate metadata.
```

It's not clear to me how "and for requests using authentication providers" relates. Is there also a 30 minute delay in metadata there? If so, the next sentence seems to contradict that.

Reply:
Removed the contradicting bit! Left the Users table part, as the engineer wanted to highlight that detail specifically to users.

Reply:
Yeah so the original message we were trying to convey was, "Each request is guaranteed to have the most up to date metadata associated with their auth provider, however changes can take up to 30 minutes to propagate to the users table and requests using other authentication providers." just to make it clear that authenticating with a _different_ authentication provider may not have the metadata specified in the request made prior with the original authentication provider used. 

With how this PR currently words this, I'd be good with going with @krollins-mdb suggestion and removing the "Users Table" part out of this as well!